core/src/main/java/com/linecorp/armeria/client/RedirectingClient.java

```diff
@@ -471,13 +472,7 @@ private static void abortResponse(HttpResponse originalRes, ClientRequestContext
 
     private static boolean isCyclicRedirects(RedirectContext redirectCtx,
                                              String redirectUri, HttpMethod method) {
-        final boolean added = redirectCtx.addRedirectUri(method, redirectUri);
-        if (!added) {
-            return true;
-        }
-
-        return redirectCtx.originalUri().equals(redirectUri) &&
-               redirectCtx.request().method() == method;
+        return !redirectCtx.addRedirectSignature(redirectUri, method);
     }
```

Comment:
What do you think about inlining this method given it is a simple delegation?

Reply:
I think it's a good idea to inline it.
The method name itself is easy to understand, so I would define it as a variable name on the caller side as below:

```java
final boolean isCyclicRedirects = !redirectCtx.addRedirectSignature(redirectUri, method);
if (isCyclicRedirects) { ... }
```